### ISSUE TITLE
Omero plugin display missing functions

### ISSUE THREAD

**Issue by jo-mueller** [tlambert03/napari-omero#40]
Hi,

when I open the Omero image browser, I get to see this view of the plugin:
![Capture](https://user-images.githubusercontent.com/38459088/178741641-f17532ea-423f-4088-9c97-61d57d78d95a.JPG)

I was wondering whether there's something missing that is not displayed here? For instance, I'm looking for an option to refresh the browser or to close the connection.

System information:

```
napari: 0.4.15
Platform: Windows-10-10.0.17763-SP0
Python: 3.9.12 (main, Apr 4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)]
Qt: 5.15.3
PyQt5: 5.15.4
NumPy: 1.22.4
SciPy: 1.8.1
Dask: 2022.7.0
VisPy: 0.9.6

OpenGL:
- GL version: 4.6.0 NVIDIA 516.59
- MAX_TEXTURE_SIZE: 32768

Screens:
- screen 1: resolution 1920x1080, scale 1.0
- screen 2: resolution 1680x1050, scale 1.0

Plugins:
- PlatyMatch: 0.0.3
- RedLionfish: 0.5
- animation: 0.0.2
- clEsperanto: 0.18.0
- console: 0.0.4
- devbio_napari: 0.5.9
- napari-3d-ortho-viewer: 0.0.1
- napari-accelerated-pixel-and-object-classification: 0.8.1
- napari-assistant: 0.3.10
- napari-brightness-contrast: 0.1.7
- napari-clusters-plotter: 0.5.1
- napari-crop: 0.1.5
- napari-curtain: 0.1.1
- napari-folder-browser: 0.1.3
- napari-layer-details-display: 0.1.4
- napari-mouse-controls: 0.1.3
- napari-plot-profile: 0.2.1
- napari-plugin-search: 0.1.3
- napari-segment-blobs-and-things-with-membranes: 0.3.0
- napari-simpleitk-image-processing: 0.4.0
- napari-tabu: 0.1.5
- napari-time-slicer: 0.4.9
- napari-tools-menu: 0.1.15
- napari-workflow-inspector: 0.2.2
- napari-workflow-optimizer: 0.1.4
- napari_skimage_regionprops1: 0.5.3
- napari_skimage_regionprops2: 0.5.3
- omero: 0.1.1
- scikit-image: 0.4.15
- svg: 0.1.6
- the-segmentation-game: 0.1.0
```

**Comment by tlambert03** [comment]
> For instance, I'm looking for an option to refresh the browser or to close the connection.


:joy: yes these would be very reasonable things to look for!  believe it or not, that's the first request though :)  will have to add those.

However, if you only have a single dataset on OMERO, that's roughly what it should look like (though, if you click, you should get a thumbnail browser).  Here's a demo of what i see on a fuller instance:

https://github.com/tlambert03/napari-omero/blob/main/demo.gif

**Comment by jo-mueller** [comment]
I implemented a disconnect button [here](https://github.com/jo-mueller/napari-omero/tree/add-disconnect-button) but I'd be curious about your feedback on that. It turned out slighty more complicated as I thought as the `OMEROTreeModel` in also connects to the gateway being opened - which led to the plugin appending multiple copies of the same project to the `QTreeView` widget after successive disconnecting/connecting.

I solved this by creating a fresh `OMEROWidget.model` everytime the plugin connects in a small helper function. I'd be happy about any feedback :)
